self_supervised_pretraining/ssl_script_train.py

```diff
@@ -0,0 +1,251 @@
+    json_Path = os.path.normpath('/scratch/data_2021/tcia_covid19/dataset_split.json')
+    data_Root = os.path.normpath('/scratch/data_2021/tcia_covid19')
+    pickle_path = os.path.normpath('/scratch/data_2021/tcia_covid19/debug_run.pkl')
+
+    # TODO Load Json & Append Root Path
+    with open(json_Path, 'r') as json_f:
+        json_Data = json.load(json_f)
+
+    train_Data = json_Data['training']
+    val_Data = json_Data['validation']
+
+    for idx, each_d in enumerate(train_Data):
+        train_Data[idx]['image'] = os.path.join(data_Root, train_Data[idx]['image'])
+
+    for idx, each_d in enumerate(val_Data):
+        val_Data[idx]['image'] = os.path.join(data_Root, val_Data[idx]['image'])
+
+    print('Total Number of Training Data Samples: {}'.format(len(train_Data)))
+    print(train_Data)
+    print('#######################')
+    print('Total Number of Validation Data Samples: {}'.format(len(val_Data)))
+    print(val_Data)
+    print('#######################')
+
+    ###############End Of Append Paths#################
+    # TODO Set Determinism
+    set_determinism(seed=453)
+
+    # TODO Define Training Transforms
+    train_Transforms = Compose(
+        [
+        LoadImaged(keys=["image"]),
+        EnsureChannelFirstd(keys=["image"]),
+        Spacingd(keys=["image"], pixdim=(
+            2.0, 2.0, 2.0), mode=("bilinear")),
+        ScaleIntensityRanged(
+            keys=["image"], a_min=-57, a_max=164,
+            b_min=0.0, b_max=1.0, clip=True,
+        ),
+        CropForegroundd(keys=["image"], source_key="image"),
+        SpatialPadd(keys=["image"], spatial_size=(96, 96, 96)),
+        RandSpatialCropSamplesd(keys=["image"], roi_size=(96, 96, 96), random_size=False, num_samples=2),
+        #RandSpatialCropd(keys=["image"], roi_size=(96, 96, 96), random_size=False),
+        #TODO Ask Wenqi, if I create another copy of the image using CopyItemsd and then pass both 'image_1', 'image_2'
+        # through the augmentation transforms, will I get the same exact augmentation or will it be different?
+        CopyItemsd(keys=["image"], times=1, names=["gt_image"], allow_missing_keys=False),
+        OneOf(transforms=[
+            RandCoarseDropoutd(keys=["image"], prob=1.0, holes=6, spatial_size=5, dropout_holes=True,
+                               max_spatial_size=32),
+            RandCoarseDropoutd(keys=["image"], prob=1.0, holes=6, spatial_size=20, dropout_holes=False,
+                               max_spatial_size=64),
+            ]
+        ),
+        RandCoarseShuffled(keys=["image"], prob=0.8, holes=10, spatial_size=8)
+        ]
+    )
+
+    check_ds = Dataset(data=train_Data, transform=train_Transforms)
+    check_loader = DataLoader(check_ds, batch_size=1)
+    check_data = first(check_loader)
+    orig_image = (check_data["gt_image"][0][0])
+    image = (check_data["image"][0][0])
+    print(f"image shape: {image.shape}")
+
+    '''
```

Comment:
Please remove these test code.

Thanks.

Reply:
This is removed.